docs/quickstarts/java-quickstart.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: docs
+title: Java Quickstart
+---
+
+<h1 class="page-header">Java Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Java with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+* `JDK`: version 7 or higher
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository (the following command clones the entire repository, but you just need the examples for this quickstart and other tutorials):
+
+```sh
+  $ # Clone the repository at the latest release to get the example code:
+  $ git clone -b v1.0.0 https://github.com/grpc/grpc-java
+  $ # Navigate to the Java examples:
+  $ cd grpc/examples
+```
+
+## Run a gRPC application
+
+From the `examples` directory:
+
+1. Compile the client and server
+
+   ```sh
+   $ ./gradle installDist
+   ```
+
+2. Run the server
+
+   ```sh
+   $ ./build/install/examples/bin/hello-world-server
+   ```
+
+3. In another terminal, run the client
+
+   ```sh
+   $ ./build/install/examples/bin/hello-world-client
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?]() and [gRPC Basics: Java][]. For now all you need
```

Comment:
Discussed with @zinuga: We can drop the reference to What is gRPC? here to simplify.